motionplan/rrtStarConnect.go

```diff
@@ -196,9 +196,9 @@ func (mp *rrtStarConnectMotionPlanner) extend(
 	minCost := math.Inf(1)
 	minIndex := -1
 	for i, neighbor := range neighbors {
-		neighborNode := neighbor.node.(*costNode)
-		cost := neighborNode.cost + neighbor.dist
-		if mp.checkPath(neighborNode.Q(), target) {
+		neighbogNode := neighbor.node.(*costNode)
```

Comment:
[nit] Typo for `neighbor` FnR really got you on this one.

Reply:
🤭